02_bubble/luliangLi/main_test.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestBubbleOutput(t *testing.T) {
+
+	testCases := map[string]struct {
+		input []int
+		want  []int
+	}{
+		"normal":     {input: []int{3, 2, 1, 5}, want: []int{1, 2, 3, 5}},
+		"blank":      {input: []int{}, want: []int{}},
+		"nil":        {input: nil, want: nil},
+		"duplicated": {input: []int{1, 0, 0, 1}, want: []int{0, 0, 1, 1}},
+	}
+
+	for name, test := range testCases {
+		// t.Run creates a sub test and runs it like a normal test
+		test := test
```

Comment:
This line is extra

Reply:
Thanks and please refer to my answer in #177